### ISSUE TITLE
Dateformatting bugs

### ISSUE THREAD

**Issue by justwin-tel** [davidquinn/psi-header#66]
When I use YYYY in the dateformat, I'm getting 232023 instead of just 2023. If I switch to yyyy, it produces 2023.

When I use DD in the dateformat, I'm not getting a 0-padded day. I tried dd, but that didn't work either.

**Comment by davidquinn** [comment]
I have done some testing and have not been able to replicate either of your problems. In all cases, YYYY produces 2023 and DD gives me the correct zero-padded day.

Are you able to share your configuration (just the psi-header parts).

Regards
D

**Comment by davidquinn** [comment]
Hi,

I have tested again with your complete configuration and I have managed to replicate what you are seeing. The difference is that the dataformat() call in the changeLogEntryFormat does not have quotes around the format string. Try modifying your config to be `"changeLogEntryTemplate": ["<<dateformat('YYYY.MM.DD')>>\t<<initials>>\t\t"]`. I notice that one of the examples also does not add quotes so I will change that and the incorrect reference to `changeLogItemTemplate`.

D
